### DISCUSSION TITLE
Is there a way to interrupt a word that is playing?

### DISCUSSION THREAD

**Discussion by bfallc** [ArminJo/Talkie#43] [Q&A]
Hey, 

I am using the Talkie library with an atmega32u4, works well so far. Is there a way to interrupt the playback from a `voice.say(...);` command part way through with a new `voice.say(...) `command? 

Thank you.

**Comment by ArminJo** [comment]
Look at the [API](https://github.com/ArminJo/Talkie/blob/master/src/Talkie.h)
